src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -32,8 +40,16 @@ const MessageContainer = styled.View`
   margin-top: ${props => props.theme.spacings.sm};
 `
 const Message = styled(HeadingBackground)`
+  color: ${ExerciseUnlocked ? prop => prop.theme.colors.primary : prop => prop.theme.colors.background};
   text-align: center;
 `
+const Icon = styled.TouchableOpacity`
+  position: absolute;
+  padding-bottom: 250px;
```

Comment:
This will only work when the device hast the correct size, but we need a solution that will also work on bigger and smaller screens. So just use `top: ??px` and `right: ??px`. Also you need to add `overflow: hidden` on the `UpperSection`. Otherwise the calculation of `right` will not work.